ppr-ui/tests/unit/test-data/mock-registration-responses.ts

```diff
@@ -29,22 +27,22 @@ export const mockedRegistration2: RegistrationSummaryIF = {
 
 export const mockedDraft1: DraftResultIF = {
   type: DraftTypes.FINANCING_STATEMENT,
-    documentId: 'D9000018',
-    baseRegistrationNumber: '',
-    registrationType: APIRegistrationTypes.REPAIRERS_LIEN,
-    registrationDescription: '',
-    path: '/path/to/doc',
-    createDateTime: '2021-08-03T17:21:17+00:00',
-    clientReferenceId: 'FFF555'
+  documentId: 'D9000018',
+  baseRegistrationNumber: '',
+  registrationType: APIRegistrationTypes.REPAIRERS_LIEN,
+  registrationDescription: '',
+  path: '/path/to/doc',
+  createDateTime: '2021-08-03T17:21:17+00:00',
+  clientReferenceId: 'FFF555'
 }
 
 export const mockedDraft2: DraftResultIF = {
   type: DraftTypes.FINANCING_STATEMENT,
-    documentId: 'D9000338',
-    baseRegistrationNumber: '',
-    registrationType: APIRegistrationTypes.MISCELLANEOUS_OTHER,
-    registrationDescription: '',
-    path: '/path/to/doc',
-    createDateTime: '2021-08-01T17:21:17+00:00',
-    clientReferenceId: ''
+  documentId: 'D9000338',
+  baseRegistrationNumber: '',
+  registrationType: APIRegistrationTypes.OTHER,
```

Comment:
Should we add a description here? I guess this begs a bigger picture question... if the user fills out the dialog with their own registration description, how does this get sent to the API?

Reply:
so when they enter the name in the modal it is saved to the store in the new *registrationTypeOtherDesc* . Doug said he would add *otherDescription* as a field in the call that the api is expecting so we can add that in to the post (I didn't touch that since this code is supposed to just be for the dropdown)

Reply:
I didn't add it to the registrationType object because it is a bit of an edge case since it only applies to 1 out of the 20 or so types